tests/mbed/mbed.edl

```diff
@@ -2,14 +2,47 @@
 // Licensed under the MIT License.
 
 enclave {
-     struct mbed_args {
+    include "fcntl.h"
+    include "sys/types.h"
+    include "sys/stat.h"
+    include "sys/uio.h"
+
+    struct mbed_args {
         int skipped;
         int total;
     };
+
+    enum string_limit {
+        STRLEN = 1024
+    };
+
     trusted {
         public int test(
             [in, string] const char* in_testname,
-            [user_check] char** out_testname,
-            [out] mbed_args* args );
+            [out, size=1024] char* out_testname,
```

Comment:
Clean implementation. 
Great that user_check was avoided.

Reply:
That fixed a leak of memory that was allocated inside the enclave and never released in the host.